x-pack/plugins/siem/public/components/open_timeline/index.tsx

```diff
@@ -190,39 +215,61 @@ export class StatefulOpenTimeline extends React.PureComponent<Props, State> {
   };
 
   /** Invoked when the user clicks the action to add the selected timelines to favorites */
-  private onAddTimelinesToFavorites: OnAddTimelinesToFavorites = () => {
-    const { addTimelinesToFavorites } = this.props;
-    const { selectedItems } = this.state;
+  // private onAddTimelinesToFavorites: OnAddTimelinesToFavorites = () => {
+  // const { addTimelinesToFavorites } = this.props;
+  // const { selectedItems } = this.state;
+  // if (addTimelinesToFavorites != null) {
+  //   addTimelinesToFavorites(getSelectedTimelineIds(selectedItems));
+  // TODO: it's not possible to clear the selection state of the newly-favorited
+  // items, because we can't pass the selection state as props to the table.
+  // See: https://github.com/elastic/eui/issues/1077
+  // TODO: the query must re-execute to show the results of the mutation
+  // }
+  // };
```

Comment:
I would just delete this chunk of code out at this point or add a clear reason why it's commented out sitting here.